Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+
+## 3. Edit (Make Edits and save as branch)
+
+1. Paste your edits into the right section. Just note that if you’re adding a ‘header’, ‘callout’ or any form of special formatted text, take a look to see how headers have been written, so you can copy the syntax. 
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Edit1.png" />
+    For example, I added a tip in the example above.   
+
+    A tip is written like this:
+     {{%callout%}}
+    This is a tip
+     {{/%callout%}}. 
+
+    Try to also keep to the indentation format so that the text looks neat and readable.
+
+    Unfortunately, at this point in time, it is not possible to preview what it will look like on the actual website, but you can click on the Preview Changes tab to see if the edits show up in the right places. 
+
+
+1. Scroll down to the Commit Changes section, and fill in the title. It defaults to ‘Update index.md’, but delete that and give it a short summary of **what you did** in less that 50 words. Refer to the image below for an example.
+1. Fill up the extended description with the reason **why** the update was needed, or the context around the update. This will be read by the reviewers - help them understand why they should approve this edit!
+1. Select ‘Create a new branch for this commit’
+1. Create a name for the new branch that explains **what it is** 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Edit2.png" />. 
+1. Click Propose Changes. 
+1. Click on the Pull Requests tab. Do NOT click the green ‘Create Pull Request’ button that appears at the bottom of the next screen.
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Edit7.png" />
+
+## 4. Submit a Pull Request (Submit edits for review)
+
+1. Click ‘New Pull Request’. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR1.png" />
+1. Make sure that your base repository is vmware-tanzu/tanzu-dev-po..., and the head repository is your account.  
+
+1. Change the dropdown for your “compare: main” to the name of the branch you have created. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR2.png" />
+1. Scroll down to check that text highlighted in green is the change you want to make.   
+1. Click Create Pull Request
```

Comment:
I think we should advise contributors to always start by including the text `WIP` (work in progress) in their pull request title. This tells the maintainers that the request is still being worked on and they they can hold off on reviewing it.

By doing so, we give the contributor some time to wait for the deploy preview to go live and do a quick self review. When the contributor is satisfied with the change and is ready for it to be reviewed they can simply edit the title to remove the `WIP`.